### PR TITLE
[oracle_weblogic] Add Integration Package with Access Logs Data Stream

### PR DESCRIPTION
**Pull request by kush-elastic**
- Enhancement

## What does this PR do?

- Generated the skeleton of Oracle WebLogic integration package.
- Added 1 data stream (Access Logs).
- Added data collection logic for the data streams.
- Added the ingest pipeline for the data streams.
- Mapped fields according to the ECS schema and added Fields metadata in the appropriate yaml files.
- Added dashboards and visualizations.
- Added system test cases for the data stream.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/oracle_weblogic directory.
- Run the following command to run tests.
`elastic-package test`

## Related issues

- Relates #2823
- Relates #2910
- Relates #3066
- Relates #3067
- Relates #3069
- Relates #3121
- Relates #3189

## Screenshots

oracle_weblogic  Access Dashboard

### CODE REVIEW

File: packages/oracle_weblogic/data_stream/access/fields/fields.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- name: oracle_weblogic
+  type: group
+  fields:
+    - name: access
+      type: group
+      fields:
+        - name: authuser
+          type: keyword
+          description: The User identity allowing them access to an online service, connected device, or other resource
+        - name: host_address
```
Comment (agithomas) [inline packages/oracle_weblogic/data_stream/access/fields/fields.yml:10]: For the other two usecases - domain log, admin server log , we use as below. Why different here?

` name: machine_name
          type: keyword
          description: Machine Name is the DNS name of the computer that hosts the server instance.`

File: packages/oracle_weblogic/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    },
+    "tags": [
+        "oracle_weblogic-access"
+    ]
+}
+```
+
+**Exported fields**
+
+| Field | Description | Type |
```
Comment (agithomas) [inline packages/oracle_weblogic/docs/README.md:107]: Can you please validate the exported field list.Do we have these number of field values available for exporting from access log, considering the dissect patter is as below. 

Request you to recheck and proceed if ok
```
pattern: '%{oracle_weblogic.access.host_address} %{} %{oracle_weblogic.access.authuser} [%{@timestamp}] "%{http.request.method}
      %{http.request.referrer} HTTP/%{http.version}" %{http.response.status_code}
```

File: packages/oracle_weblogic/kibana/lens/oracle_weblogic-40a02e00-b895-11ec-b6f8-0795d197226b.json [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+                                        "size": 10
+                                    },
+                                    "scale": "ordinal",
+                                    "sourceField": "oracle_weblogic.access.host_address"
+                                },
+                                "874819de-f117-481a-ba1a-3324eade0bb3": {
+                                    "customLabel": true,
+                                    "dataType": "string",
+                                    "isBucketed": true,
+                                    "label": "Request Endpoint",
```
Comment (agithomas) [inline packages/oracle_weblogic/kibana/lens/oracle_weblogic-40a02e00-b895-11ec-b6f8-0795d197226b.json:42]: Suggestion: When i checked the example, what is shown in word cloud is asset / resource. If endpoint, better be a non-resource, eliminating imgs/ js/ ico . You may check this with PM

File: packages/oracle_weblogic/data_stream/access/fields/fields.yml (hunk 1 above)
Comment (kush-elastic) [reply packages/oracle_weblogic/data_stream/access/fields/fields.yml:10]: In the `Domain`, `Admin server` and `Managed server` we receive `Machine name` that gives DNS name of the computer and not the Host IP. However In the HTTP Access Logs we may receive Host IP/ DNS name of the Host. 

So we are using `host_address` in `access logs` and `machine_name` in other three. What do you suggest?

File: packages/oracle_weblogic/kibana/lens/oracle_weblogic-40a02e00-b895-11ec-b6f8-0795d197226b.json (hunk 3 above)
Comment (kush-elastic) [reply packages/oracle_weblogic/kibana/lens/oracle_weblogic-40a02e00-b895-11ec-b6f8-0795d197226b.json:42]: Can you elaborate this from where should I refer to?

Comment (agithomas) [reply packages/oracle_weblogic/kibana/lens/oracle_weblogic-40a02e00-b895-11ec-b6f8-0795d197226b.json:42]: I think we can take it up in the next iteration. 
It is just a recommendation.

**Review by agithomas — COMMENTED** [review]
Change looks good to me

**Review by agithomas — APPROVED** [review]
Looks good to me

**Review by yug-elastic — APPROVED** [review]

**Pull request merged**